app/services/users.js

```diff
@@ -27,3 +27,13 @@ exports.createUser = async ({ name, lastName, email, password }) => {
     throw databaseError('Error signUp user');
   }
 };
+
+exports.getUsers = async ({ limit, offset }) => {
```

Comment:
As a suggestion, this could be returned directly without the need for the try.


something like this

exports.getUsers = pagination =>
  User.findAndCountAll({
  }).catch(error => {
  });